src/Components/SmartComponents/Reports/Common/tablePage.js

```diff
@@ -3,13 +3,14 @@ import PropTypes from 'prop-types';
 import { View, Text, Link } from '@react-pdf/renderer';
 import { Table } from '@redhat-cloud-services/frontend-components-pdf-generator';
 import { processDate } from '@redhat-cloud-services/frontend-components-utilities/helpers';
+import { parseRhelVersions } from '../../../../Helpers/VulnerabilityHelper';
 import { CVES_PATH } from '../../../../Helpers/constants';
 import messages from '../../../../Messages';
 import styles from './styles';
 import { formatDate } from '../../../../Helpers/MiscHelper';
 import CVElabels from '../Common/CVElabels';
 
-const tablePage = ({ data, page, intl, header, type, isReportDynamic = false }) => {
+const tablePage = ({ data, page, intl, header, type, isReportDynamic = false, isAppliesToOSEnabled }) => {
```

Comment:
Tiny nitpick. Since this returns JSX it should be a component and rather be `TablePage`.

Reply:
aha, makes sense! fixing